First Mod Passed

lakishaJohnson opened the pull request.

Angiepmunoz commented on index.js:
```diff
@@ -142,7 +240,46 @@ function getAllMoviesReleasedAtOrBeforeYear() {}
+
+for (let movie of movies){
+  
+  let moneyMade = 1 * (movie.boxOffice.slice(1).split(',').join(''))
```
great job

Angiepmunoz commented on index.js:
```diff
@@ -142,7 +240,46 @@ function getAllMoviesReleasedAtOrBeforeYear() {}
+  if (moneyMade > biggestMovie){
+    biggestMovie = moneyMade
+    
+    smashHit = movie.title
```
where do you declared the variable smashHit?

lakishaJohnson commented:
Great question, that I looked over and was a bit confused as to how that passed. I feel like I may have hard coded that. But I had a whole other code where 3/4 passed and when I moved that line, it worked. I didn't want to break my code again so I just left it. I changed this code so many times I may be confused about it myself. Will you please explain it to me?Sent from the all new AOL app for Android 
 
  On Mon, Aug 16, 2021 at 12:14, ***@***.***> wrote:   


@Angiepmunoz commented on this pull request.

In index.js:
> @@ -142,7 +240,46 @@ function getAllMoviesReleasedAtOrBeforeYear() {}
  *  getBiggestBoxOfficeMovie(movies);
  *  //> "Incredibles 2"
  */
-function getBiggestBoxOfficeMovie() {}
+function getBiggestBoxOfficeMovie(movies) {
+ if (movies.length === 0){
+  return null;
+}
+
+let biggestMovie = 0
+
+for (let movie of movies){
+  
+  let moneyMade = 1 * (movie.boxOffice.slice(1).split(',').join(''))

great job

In index.js:
> +  return null;
+}
+
+let biggestMovie = 0
+
+for (let movie of movies){
+  
+  let moneyMade = 1 * (movie.boxOffice.slice(1).split(',').join(''))
+  //MULTIPLY STRING BY ONE TO CHANGE IT INTO A NUMBER
+  //.SLICE USED TO REMOVE $ SIGN
+  //.SPLIT USED TO REMOVE COMMA'S
+  //.JOIN USED TO REMOVE SPACES LEFT BY COMMA'S
+  if (moneyMade > biggestMovie){
+    biggestMovie = moneyMade
+    
+    smashHit = movie.title

where do you declared the variable smashHit?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

lakishaJohnson commented:
I see it... Its in an old code further down at the bottom.

Sent from the all new AOL app for Android 
 
  On Mon, Aug 16, 2021 at 12:14, ***@***.***> wrote:   


@Angiepmunoz commented on this pull request.

In index.js:
> @@ -142,7 +240,46 @@ function getAllMoviesReleasedAtOrBeforeYear() {}
  *  getBiggestBoxOfficeMovie(movies);
  *  //> "Incredibles 2"
  */
-function getBiggestBoxOfficeMovie() {}
+function getBiggestBoxOfficeMovie(movies) {
+ if (movies.length === 0){
+  return null;
+}
+
+let biggestMovie = 0
+
+for (let movie of movies){
+  
+  let moneyMade = 1 * (movie.boxOffice.slice(1).split(',').join(''))

great job

In index.js:
> +  return null;
+}
+
+let biggestMovie = 0
+
+for (let movie of movies){
+  
+  let moneyMade = 1 * (movie.boxOffice.slice(1).split(',').join(''))
+  //MULTIPLY STRING BY ONE TO CHANGE IT INTO A NUMBER
+  //.SLICE USED TO REMOVE $ SIGN
+  //.SPLIT USED TO REMOVE COMMA'S
+  //.JOIN USED TO REMOVE SPACES LEFT BY COMMA'S
+  if (moneyMade > biggestMovie){
+    biggestMovie = moneyMade
+    
+    smashHit = movie.title

where do you declared the variable smashHit?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
